nushell/nushell
# Ctrl+c interruption - `cp` command

# Description
if you try to copy a big file with `cp` you will noticed that you can't interrupt the process. This pull request fix that.
This was discuss here https://github.com/nushell/nushell/pull/8012#issuecomment-1427313054

# User-Facing Changes
None

# Tests + Formatting
- Check -   `cargo fmt --all -- --check` to check standard code formatting (`cargo fmt --all` applies these changes)
- Check -  `cargo clippy --workspace -- -D warnings -D clippy::unwrap_used -A clippy::needless_collect` to check that you're using the standard code style
- Check -  `cargo test --workspace` to check that all tests pass
